app/shell-window/pages.js

```diff
@@ -220,6 +223,16 @@ export function create (opts) {
       navbar.update(page)
     },
 
+    setFullscreen (isFullscreen) {
```

Comment:
Assuming the `fullscreenWarning` global isn't necessary, then this page method (setFullscreen) isn't necessary either. We can just use `promptbar.add` and `.remove` directly in `shell-window/ui.js`. (The page object is already really big so I prefer to avoid adding new methods to it.)